depmapomics/qc/conftest.py

```diff
@@ -1,5 +1,6 @@
+from depmapomics import constants
```

Comment:
imported but unused, same for all other modules in depmapomics/qc

Reply:
removed